netlify.toml

```diff
@@ -1,147 +1,123 @@
-# Settings in the [build] context are global and are applied to all contexts
-# unless otherwise overridden by more specific contexts.
 [build]
-  # Directory to change to before starting a build.
-  # This is where we will look for package.json/.nvmrc/etc.
-  base = ""
-
   # Directory (relative to root of your repo) that contains the deploy-ready
-  # HTML files and assets generated by the build. If a base directory has
-  # been specified, include it in the publish directory path.
+  # HTML files and assets generated by the build.
   publish = "website/build/testing-library-docs"
 
   # Default build command.
-  command = "cd website; npm install; npm run format-docs; npm run build"
+  command = "npm i -g yarn; cd website; yarn; yarn format-docs; yarn build"
```

Comment:
`yarn` should already be installed in netlify. If you need a specific version you can check it into the repo with [`yarn set policies`](https://classic.yarnpkg.com/en/docs/cli/policies/#toc-policies-set-version)

Reply:
I think that's true only if the root folder contains a `yarn.lock` file, since I see here that we `cd website`, it looks like the root folder isn't configured properly in netlify so maybe that's the reason why we needed the installation.

Reply:
That's an odd design decision though I can see why you'd want to do this anyway. For apps you should always have a lockfile. I think we just caried over the no-lockfile policy from the lib repos.

Reply:
I actually didn't even remember we don't have a lock file.. But that explains the need for global `yarn` installation.

Reply:
I can confirm. I tested this out without manually installing Yarn and got errors that it wasn't installed (since we haven't committed `yarn.lock`). Alternatively it may make more sense to just switch to npm, but I thought it would be best to keep things simple for this PR.

Reply:
Out of curiosity let's have a vote on keeping vs removing a lockfile for this repo (since it's a site not a package).

👍  Commit `yarn.lock` or switch to npm for consistently and commit `package-lock.json`
👎  Keep ignored and possibly switch to npm since we're not sharing `yarn.lock`